src/app/containers/Timestamp/index.test.jsx

```diff
@@ -38,46 +42,32 @@ describe('Timestamp', () => {
     />,
   );
 
-  it('should display only one timestamp when published === updated', () => {
+  it('should render only one timestamp when published === updated', () => {
```

Comment:
```suggestion
  it('should render one date when published before today and never updated', () => {
```

I'd move this test below the next one too.